src/test/java/org/tub/vsp/bvwp/computation/ComputationKNTest.java

```diff
@@ -0,0 +1,38 @@
+package org.tub.vsp.bvwp.computation;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.tub.vsp.bvwp.computation.ComputationKN.nkv;
+
+class ComputationKNTest {
+    @Test
+    @Disabled
+        //needs to be fixed
```

Comment:
@kainagel der Test ist nun in einer eigenen Klasse. Allerdings schlägt er fehlt - vermutlich noch wegen alter Daten und Veränderung in der Berechnung.